Cargo.toml

```
@@ -79,7 +79,7 @@ ring = ["dep:ring"]
 std = ["alloc", "pki-types/std"]
 
 [dependencies]
-aws-lc-rs = { version = "1.0.0", optional = true }
+aws-lc-rs = { version = "1.6", optional = true, default-features = false, features = ["aws-lc-sys"] }
```

Yeah I suggest we do:

```suggestion
aws-lc-rs = { version = "1", optional = true, default-features = false, features = ["aws-lc-sys"] }
```

---

changed